llvm/lib/FuzzMutate/FuzzerCLI.cpp

```diff
@@ -107,7 +108,24 @@ void llvm::handleExecNameEncodedOptimizerOpts(StringRef ExecName) {
       Args.push_back("-passes=loop-reduce");
```

Comment:
Not directly related to this PR, but I noticed `-passes=loop(rotate)` up there, which doesn't make sense. This should be `-passes=loop(loop-rotate)` or just `-passes=loop-rotate`.

Reply:
Fixed -- set it to `-passes=loop-rotate`